test/e2e/workflow/deployment_test.go

```diff
@@ -160,6 +169,26 @@ var _ = Describe("NetworkAddonsConfig", func() {
 			CheckConfigCondition(ConditionAvailable, ConditionTrue, 10*time.Minute, 30*time.Second)
 		})
 	})
+
+	//TODO: remove Eventually and just use GetConfig after issue 141 is resolved
+	// (https://github.com/kubevirt/cluster-network-addons-operator/issues/141)
+	Context("when restarting kubeMacPool", func() {
+		It("should generate a new range that is different than the old one", func() {
+			config := opv1alpha1.NetworkAddonsConfigSpec{KubeMacPool: &opv1alpha1.KubeMacPool{}}
```

Comment:
Last one, all this code before Expect should be put at a BeforeEach, it's going to behave the same but will make sense with Context

Take a look here https://medium.com/swlh/effective-ginkgo-gomega-b6c28d476a09